clients/python-client/python_client/kuberay_cluster_api.py

```diff
@@ -32,9 +32,17 @@ class RayClusterApi:
     # initial config to setup the kube client
     def __init__(self):
         # loading the config
-        self.kube_config: Optional[Any] = config.load_kube_config()
+        try:
+            self.kube_config: Optional[Any] = config.load_kube_config()
+        except config.ConfigException:
+            # No kubeconfig found, try in-cluster config
+            try:
+                self.kube_config: Optional[Any] = config.load_incluster_config()
+            except config.ConfigException:
+                log.error("Failed to load both kubeconfig and in-cluster config")
+                raise
+
         self.api = client.CustomObjectsApi()
-        self.core_v1_api = client.CoreV1Api()
```

Comment:
Isn't this used by some example files?

Reply:
@rueian Ah you're correct, added it back, thanks :)